rts/Sim/Units/CommandAI/BuilderCAI.cpp

```diff
@@ -1149,8 +1149,10 @@ void CBuilderCAI::ExecuteResurrect(Command& c)
 
 				if (ownerBuilder->lastResurrected && unitHandler.GetUnitUnsafe(ownerBuilder->lastResurrected) != nullptr && owner->unitDef->canRepair) {
 					// resurrection finished, start repair (by overwriting the current order)
-					c = Command(CMD_REPAIR, c.GetOpts() | INTERNAL_ORDER, ownerBuilder->lastResurrected);
-
+					float3 pos = c.GetPos(1);
+					float radius = c.GetParam(4);
+					c = Command(CMD_REPAIR, c.GetOpts() | INTERNAL_ORDER, ownerBuilder->lastResurrected, pos);
+					c.PushParam(radius);
```

Comment:
Only add leash if the original command has it:
```suggestion
					if (c.GetNumParams() == 5) { // cruel leash
						float3 pos = c.GetPos(1);
						float radius = c.GetParam(4);
						c = Command(CMD_REPAIR, c.GetOpts() | INTERNAL_ORDER, ownerBuilder->lastResurrected, pos);
						c.PushParam(radius);
					} else {
						c = Command(CMD_REPAIR, c.GetOpts() | INTERNAL_ORDER, ownerBuilder->lastResurrected);
					}
```

Reply:
Done